source/runtime.cpp

```diff
@@ -859,6 +860,9 @@ void reshade::runtime::load_config()
 #endif
 
 	config.get("SCREENSHOT", "SavePath", _screenshot_path);
+	config.get("SCREENSHOT", "SoundPath", _screenshot_playsound_path);
+	config.get("SCREENSHOT", "PlayDefaultIfNotExist", _screenshot_playsound_force);
+	config.get("SCREENSHOT", "PlaySoundAsSystemNotification", _screenshot_playsound_as_system_notification);
```

Comment:
I'm usually all for configurability, but in this case I'm questioning whether we really need these two extra options.
It is probably fine to always play the default ding sound if the file cannot be found (can avoid the additional file exists in our code that way) and is probably also sufficient to never play in the system notification channel?

Reply:
ReShade can not adjust the volume, so we should use global volume mixer. Well, not everyone need it.

Reply:
Users can specify a non-playable file for play the default sound, but the user control does not allow it. So I removed the behavior.